pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
 ]
 requires-python = ">= 3.11, < 4"
 dependencies = [
-  "frequenz-api-microgrid >= 0.11.0, < 0.12.0",
+  "frequenz-api-microgrid >= 0.15.1, < 0.16.0",
```

Comment:
I guess the PR title needs to be updated :)

Reply:
You mean the version in the PR title?  I did that 2 days ago.  Maybe you opened the page before that and haven't refreshed?

Reply:
Nah, I just misread this and thought we were updating to v0.16.0 :facepalm: